File: ohcommunicator/src/main/java/se/treehou/ng/ohcommunicator/connector/models/ThingType.java
package se.treehou.ng.ohcommunicator.connector.models;

import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class ThingType {

    private List<Channel> channels = new ArrayList<>();
    private String description;
    private String label;

    @SerializedName("UID")
    private String uID;

    private boolean bridge;

    public List<Channel> getChannels() {
        return channels;
    }

    public String getDescription() {
        return description;
    }

    public String getLabel() {
        return label;
    }

    public String getuID() {
        return uID;
    }

    public boolean isBridge() {
        return bridge;
    }
}

File: mobile/src/main/java/treehou/se/habit/connector/models/ThingType.kt
package treehou.se.habit.connector.models

import com.google.gson.annotations.SerializedName

class ThingType {

    val channels: List<Channel>? = null
    val description: String? = null
    val label: String? = null

    @SerializedName("UID")
    private val uID: String? = null

    val isBridge: Boolean = false

    fun getuID(): String? {
        return uID
    }
}
